docs/reference/dev_help/api_standalone.md

```diff
@@ -68,14 +79,20 @@ class MyOperator(SOperator):
         super(self.__class__, self).execute(force)
 ```
 
-The example operator's constructor method takes care of initalizing the plug objects and adding them to the operator. Each plug must be initialized specifying its type (which in this case is `SType.kInt`) and a direction which can be either `SDirection.kIn` or `SDirection.kOut`.
+The `SOperator` constructor method takes care of initializing the plug objects and adding them to the operator. Each plug must be initialized specifying:
+- a unique code name, which will correspond to the name displayed in the UI
+- a value, which will correspond to the value the plug is initialized with
+- a type, which will define the behaviour of the plug when translating the provided inputs to a Python object
```

Comment:
Should we maybe list all available types here?

Reply:
This should be available in the API right?

Reply:
Yes, maybe then we should just add a todo to have a link here to point to that?

Reply:
At the moment the link can't be added, as the page does not exist and we do not know exactly how doxygen will structure it.